npc/custom/warper.txt

```diff
@@ -257,7 +258,7 @@ F23: setarray @c[0],208,227,190,206,240,206,190,143,307,252,239,213,185,188,193,
 F24: Restrict("Pre-RE",2,7,9,10,11,13);
 	 setarray @c[2],192,162,235,166,202,206,202,208,225,202,202,214,263,196,217,201,87,121,277,181,221,185,175,200,174,197;
 	 Disp("Rachel Field",1,13); Pick("ra_fild");
-F25: if(.Satan_Morocc){
+F25: if(.Satan_Morocc=1){
```

Comment:
This is an assignment and no compare now.
Therefore this does not fix something, rather break something.